src/pages/LookupConfig.page

```diff
@@ -1,3 +1,44 @@
-<apex:page showHeader="true" sidebar="true">
-	
-</apex:page>
\ No newline at end of file
+<apex:page controller = "LookupConfigController">
+	<apex:form id = "formatAddressLookupFields">
+		<apex:pageBlock title="My Content" mode ="edit">
+  		<apex:pageBlockSection title ="My Content Section" columns="2">
+				<apex:pageBlockSectionItem>
+					<apex:outputLabel value = "County Field" for = "county_field"/>
+					<apex:selectList value = "{!countyField}" multiselect = "false" size = "1" id = "county_field">
+					</apex:selectList>
```

Comment:
We'll need to add a `<apex:selectOptions>` tag here (similar to line 13, but using the `fieldList` variable instead of `objectList`.  Additionally, since we don't have those values before an object is chosen, we'll need to set the initial value of the `fieldList` variable to equal to a `new List<SelectOption>()` in the constructor of the controller.  You could easily put it right before you call `populateObjectList()` on line 14 of the controller.